rts/Sim/MoveTypes/HoverAirMoveType.cpp

```diff
@@ -410,7 +410,7 @@ void CHoverAirMoveType::UpdateHovering()
 
 	// randomly drift (but not too far from goal-position)
 	wantedSpeed = (randomWind * math::fabs(owner->unitDef->dlHoverFactor) * 0.5f);
-	wantedSpeed += (smoothstep(0.0f, 20.0f * 20.0f, (goalPos - owner->pos)) * (goalPos - owner->pos));
+	wantedSpeed += (goalPos - owner->pos) * 0.1f;
```

Comment:
It just causes the random wind to blow it a bit more towards goalPos than away from it.
How do you think this can overshoot?